tests/telemetry/test_telemetry.py

```diff
@@ -611,10 +611,10 @@ def test_log_call_add_payload_success(mock_telemetry):
     _telemetry = telemetry.Telemetry(MOCK_API_KEY, "some-package", "0.14.0")
 
     @_telemetry.log_call("some-action", payload=True)
-    def my_function(payload):
+    def my_function(payload, x, y):
         payload["dag"] = "value"
 
-    my_function()
+    my_function(1, 2)
```

Comment:
any reason to update this test? looks like it shouldn't be affected by this change

Reply:
Fixed